egograph/backend/dependencies.py

```
@@ -72,3 +78,27 @@ def get_db_connection(
 
     with DuckDBConnection(config.r2) as conn:
         yield conn
+
+
+def get_spotify_repository(
+    config: BackendConfig = Depends(get_config),
```

Ruff (0.15.13)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)